main.js

```diff
@@ -1,121 +1,125 @@
+    for (let i = discs; i > 0; i--) {
       firstPeg.push(i);
       winningPegVal += i; // 6
     }
-    
-  
+
     // ************** FUNCTIONS ***************** //
-  
+
     // Draw board
     const drawBoard = () => {
       board.map(peg => {
         let drawPeg = `----- `;
-    
+
         peg.map(disc => {
           drawPeg += `${disc} `;
         });
-    
+
         console.log(drawPeg);
       });
     };
-  
+
     drawBoard(board);
-  
+
     // returns bool whether the move is allowed or not
     const checkMove = (startPeg, endPeg) => {
       // Take disc off starting peg
       let disc = startPeg[startPeg.length - 1];
-  
+
       // Determine possible endPegs disc can move to
       let possibleEndPegs = board.filter(peg => {
         return peg.length === 0 || disc < peg[peg.length - 1];
       });
-  
+
       // if endPeg is a possible endPeg, allow the move
       return possibleEndPegs.includes(endPeg);
     };
-  
+
     // Move a disc
     const moveDisc = (startPeg, endPeg) => {
       // Arrays are zero-base indexed
       startPegArr = board[startPeg - 1];
       endPegArr = board[endPeg - 1];
-  
+
       // Take disc off starting peg
       let disc = startPegArr[startPegArr.length - 1];
-  
+
       // Check if move is allowed
       let moveAllowed = checkMove(startPegArr, endPegArr);
       // If so, move disc and increment number of moves taken
-      if(moveAllowed) {
+      if (moveAllowed) {
         console.log(`That move was successful, board is now:`);
         startPegArr.pop();
         endPegArr.push(disc);
         numMoves++;
       } else {
-        console.log(`You can't move a larger disc on top of a smaller one, board is still:`);
+        console.log(
+          `You can't move a larger disc on top of a smaller one, board is still:`
+        );
       }
-      
+
       // Redraw board and check for winner
       drawBoard(board);
       console.log(`Moves: ${numMoves}`);
-      checkWinner(endPegArr); 
+      checkWinner(endPegArr);
     };
-  
+
     // Check to see if the game has been won
-    const checkWinner = (endPeg) => {
+    const checkWinner = endPeg => {
       // Calc val of endPeg discs to compare against winning val
       let endPegVal = endPeg.reduce((acc, disc) => {
         acc += disc;
         return acc;
       }, 0);
-  
+
       // Winning conditions
-      if(endPeg !== firstPeg && endPegVal === winningPegVal) {
+      if (endPeg !== firstPeg && endPegVal === winningPegVal) {
         gameWon = true;
       }
-  
-      return gameWon;    
+
+      return gameWon;
     };
-  
-  
+
     // ***************** PLAY GAME ********************* //
     while (!gameWon) {
-      let startPeg = parseInt(prompt(`Which peg would you like to remove a disc from?`));
-      let endPeg = parseInt(prompt(`Which peg would you like to move the disc to?`))
-  
+      let startPeg = parseInt(
+        prompt(`Which peg would you like to remove a disc from?`)
```

Comment:
Right now because of the prompt-centered functionality, your game is difficult to test/play with. Next time I'd recommend either having a console game or one with a UI rather than forcing the user to play via cycle of endless prompts with no visual of the moves they're making.